bagit.py

```diff
@@ -751,8 +752,7 @@ def _make_manifest(manifest_file, data_dir, processes, algorithm='md5'):
             num_files += 1
             total_bytes += byte_count
             manifest.write("%s  %s\n" % (digest, _encode_filename(filename)))
-        manifest.close()
```

Comment:
Did you mean to remove manifest.close()? I believe it is good practice to keep it in there.